otter/models/intents.py

```diff
@@ -1,29 +1,38 @@
 from functools import partial
 
+import attr
+
 from characteristic import attributes
 
-from effect import TypeDispatcher
+from effect import TypeDispatcher, sync_performer
 
 from twisted.internet.defer import inlineCallbacks, returnValue
 
 from txeffect import deferred_performer
 
+from otter.models.cass import CassScalingGroupServersCache
+from otter.util.fp import assoc_obj
+
 
-@attributes(['scaling_group', 'modifier'])
-class ModifyGroupState(object):
+@attr.s
+class ModifyGroupStateActive(object):
```

Comment:
Well, it would make tests a lot easier, since they no longer need to deal with a callable, instead just doing an equality check on `active`, so that's nice!